py3status/module.py

```diff
@@ -350,6 +350,35 @@ def set_module_options(self, module):
 
             self.i3s_module_options['separator_block_width'] = sep_block_width
 
+        for name in ('background', 'border'):
+            if name in mod_config:
+                param = mod_config[name]
+                if not isinstance(param, basestring):
+                    raise TypeError(
+                        'Invalid attribute `{}`, '
+                        'should be an int, not `{}`'
+                        ''.format(name, param)
+                    )
+
+                self.py3_module_options['block'][name] = (
+                    Py3(self)._get_color(param)
+                )
```

Comment:
what's going on here? you need to have comments explaining stuff.  This is changes to core py3status they need to be maintainable.

Any PR should leave the code nicer than before :)

Reply:
This is same as before, except that I'm putting everything inside `self.py3_module_options = {'block': {}}` instead of (2) separate parameters.